Resources/Prototypes/_NF/Entities/Structures/Storage/Crates/base_structurecrates.yml

```diff
@@ -6,7 +6,7 @@
   - type: TradeCrate
   - type: Lock
   - type: AccessReader
-    access: [[""]]
+#    access: [[""]]
```

Comment:
This was intended, they are not even a real storage entity, as you cannot store a single item in it even if in some bug you managed to open it.